### ISSUE TITLE
RIDE listener is not working when running tests with Maven

### ISSUE THREAD

**Issue by Hi-Fi** [Hi-Fi/robotframework-java-example#1]
Tests pass fine, so this is just for seeing what tests are doing while run is ongoing.

Error printed to log is:
```
[ ERROR ] Taking listener '/Library/Python/2.7/site-packages/robotide/contrib/testrunner/TestRunnerAgent.py:57981:False' into use failed: Importing listener '/Library/Python/2.7/site-packages/robotide/contrib/testrunner/TestRunnerAgent.py' failed: ImportError: No module named robotide
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/robotide/contrib/testrunner/TestRunnerAgent.py", line 74, in <module>
    from robotide.utils import PY2, PY3
PYTHONPATH:
  /Users/juhosaarinen/.m2/repository/org/robotframework/robotframework/3.1.1/Lib
  /Users/juhosaarinen/.m2/repository/org/robotframework/robotframework/3.1.1/robotframework-3.1.1.jar/Lib
  __classpath__
  __pyclasspath__/
CLASSPATH:
  /Users/juhosaarinen/git/robotframework-java-example/.mvn/wrapper/maven-wrapper.jar
```

**Comment by Hi-Fi** [comment]
If adding `/Library/Python/2.7/site-packages` to RIDE's Pythonpath, error becomes:
```
[ ERROR ] Execution stopped by user.
[ ERROR ] Taking listener '/Library/Python/2.7/site-packages/robotide/contrib/testrunner/TestRunnerAgent.py:57981:False' into use failed: Importing listener '/Library/Python/2.7/site-packages/robotide/contrib/testrunner/TestRunnerAgent.py' failed: AttributeError: 'module' object has no attribute '__version__'
Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/robotide/contrib/testrunner/TestRunnerAgent.py", line 74, in <module>
    from robotide.utils import PY2, PY3
  File "/Library/Python/2.7/site-packages/robotide/__init__.py", line 45, in <module>
    import wx
  File "/Library/Python/2.7/site-packages/robotide/__init__.py", line 45, in <module>
    import wx
  File "/Library/Python/2.7/site-packages/wx/__init__.py", line 12, in <module>
    __version__ = wx.__version__.VERSION_STRING
PYTHONPATH:
  /Library/Python/2.7/site-packages
  /Library/Python/2.7/site-packages
  /Users/juhosaarinen/.m2/repository/org/robotframework/robotframework/3.1.1/Lib
  /Users/juhosaarinen/.m2/repository/org/robotframework/robotframework/3.1.1/robotframework-3.1.1.jar/Lib
  __classpath__
  __pyclasspath__/
CLASSPATH:
  /Users/juhosaarinen/git/robotframework-java-example/.mvn/wrapper/maven-wrapper.jar
```

**Comment by Hi-Fi** [comment]
This is fixed in RIDE 1.7.3.1.
